src/index.js

```diff
@@ -193,10 +193,14 @@ class Dropzone extends React.Component {
     const { onClick, disableClick } = this.props
     if (!disableClick) {
       evt.stopPropagation()
-      this.open()
+
       if (onClick) {
         onClick.call(this, evt)
       }
+
+      setTimeout(() => {
```

Comment:
Could you please add a comment referring to the issue and describing why this is needed?